packages/utils/src/line-items/prepare-line-item-data.ts

```diff
@@ -0,0 +1,56 @@
+import { ProductVariantDTO } from "@medusajs/types"
+import { MedusaError } from "medusa-core-utils"
+
+import pick from "lodash/pick"
+
+const VARIANT_PROPERTIES = [
+  "title",
+  "metadata",
+  "product_id",
+  "product.title",
+  "product.discountable",
+  "product.is_giftcard",
+]
+
+export type LineItemCreateData = {
+  variant_id: string
+  title: string
+  product_id: string
+  product: {
+    title: string
+    thumbnail: string | null
+    discountable: boolean
+    is_giftcard: boolean
+  }
+  metadata?: Record<string, unknown>
+
+  quantity: number
+  unit_price?: number
+}
+
+/**
+ * Extract from data need to generate line item from a variant.
+ *
+ * @param variantData
+ * @param quantity
+ * @param unit_price
+ */
+export function prepareLineItemData(
```

Comment:
todo: I would expect an object argument to make it more explicit what is expected instead of calling `fn(data, 1, 2)` it would be `fn({ variantData, quantify: 1, unit_price: 2})` this is for a comprehensive API of the functions we are building in those cases. wdyt?

Reply:
I think that is more or less the same if we have less than 5 args. Not sure that this is less comprehensive since we have argument names and types.

Reply:
let's keep it as is for now